### PR TITLE
Adding events for 2023-02-01

### PR DESCRIPTION
**Pull request by andrewpollack**

### CODE REVIEW

File: draft/2023-02-01-this-week-in-rust.md [hunk 1]
```diff
@@ -198,14 +192,41 @@ Rusty Events between 2023-02-01 - 2023-03-01 🦀
...
+* 2023-02-15 | Trondheim, NO | [Rust Trondheim](https://www.meetup.com/rust-trondheim)
+    * [**Rust New Year's Resolution Bug Hunt**](https://www.meetup.com/rust-trondheim/events/290889889/)
+* 2023-02-16, 2023-02-17 | London, UK | [Rust Nation UK](https://www.rustnationuk.com/)
+    * [**Rust Nation '23**](https://www.rustnationuk.com/)
+* 2023-02-18 | London, UK | [Rust London User Group](https://www.meetup.com/rust-london-user-group/)
+    * [**Post-Conference Rust in Enterprise Brunch Hosted at Red Badger**](https://www.meetup.com/rust-london-user-group/events/291297886/)
 * 2023-02-21 | Zurich, CH | [Rust Zurich](https://www.meetup.com/rust-zurich/)
     * [**Practical Cryptography - February Meetup (Registration opens 7 Feb 2023)**](https://www.meetup.com/de-DE/rust-zurich/events/290915075/)
+* 2023-02-23 | Copenhagen, DK | [Copenhagen Rust Community](https://www.meetup.com/copenhagen-rust-community)
+    * [**Rust metup #33**](https://www.meetup.com/copenhagen-rust-community/events/291288154/)
```
Comment (mariannegoldin) [inline draft/2023-02-01-this-week-in-rust.md:208]: Consider adding on 2/23: https://www.meetup.com/java-user-group-hessen-jugh/events/290346591/

Comment (mariannegoldin) [reply draft/2023-02-01-this-week-in-rust.md:208]: Correction: this should probably be added to the "virtual" section

File: draft/2023-02-01-this-week-in-rust.md [hunk 2]
```diff
@@ -178,18 +166,24 @@ Rusty Events between 2023-02-01 - 2023-03-01 🦀
...
 * 2023-02-15 | Virtual (Vancouver, BC, CA) | [Vancouver Rust](https://www.meetup.com/vancouver-rust/)
     * [**Rust Study/Hack/Hang-out**](https://www.meetup.com/vancouver-rust/events/tqvhxsyfcdbtb/)
+* 2023-02-21 | Virtual (Washington, DC, US) | [Rust DC](https://www.meetup.com/rustdc/)
+    * [**Mid-month Rustful**](https://www.meetup.com/rustdc/events/vdhxgsyfcdbcc/)
+* 2023-02-23 | Virtual (Charlottesville, VA, US) | [Charlottesville Rust Meetup](https://www.meetup.com/charlottesville-rust-meetup/)
+    * [**Tock, a Rust based Embedded Operating System**](https://www.meetup.com/charlottesville-rust-meetup/events/291248593/)
+* 2023-02-28 | Virtual (Dallas, TX, US) | [Dallas Rust](https://www.meetup.com/Dallas-Rust/)
+    * [**Last Tuesday**](https://www.meetup.com/dallas-rust/events/qndgwsyfcdblc/)
+* 2023-03-01 | Virtual (Indianapolis, IN, US) | [Indy Rust](https://www.meetup.com/indyrs/)
```
Comment (mariannegoldin) [inline draft/2023-02-01-this-week-in-rust.md:175]: Add: 2/28 Rust Hack and Learn - https://www.meetup.com/opentechschool-berlin/events/290852327/

**Review by mariannegoldin — APPROVED** [review]
Looks great! Approving with one suggested addition.

File: draft/2023-02-01-this-week-in-rust.md (hunk 2 above)
Comment (andrewpollack) [reply draft/2023-02-01-this-week-in-rust.md:179]: Done!

**Comment by andrewpollack** [comment]
Thank you @mariannegoldin ! Nice finds on those two -- included both

**Pull request merged**